src/sentry/quotas/base.py

```diff
@@ -491,3 +496,34 @@ def get_transaction_sampling_tier_for_volume(
         :param organization_id: The organization id.
         :param volume: The volume of transaction of the given project.
         """
+
+    def assign_seat(
+        self,
+        monitor: Monitor,
+    ) -> int:
```

Comment:
Feels weird that tis is so specific to monitors but it's generically calld `assign_seat` and is part of a generic `Quotas` service

Reply:
Until we have more kinds of seats it is hard to have an abstraction that isn't going to be wrong though.

Reply:
Yeah I agree. Maybe we could start with calling it `assign_monitor_seat`